src/pages/Withdrawals/index.tsx

```diff
@@ -0,0 +1,1064 @@
+      })}
+    >
+      <ComponentStyles>
+        <section>
+          <Text className="mb10">
+            <FormattedMessage
+              defaultMessage="The Shanghai/Capella upgrade enabled Beacon Chain staking
+              withdrawals, with the inclusion of {eip}. This Ethereum improvement proposal
+              provides a mechanism of continuously distributing rewards to stakers, and also
+              providing a way for exited validators to unlock their entire balance—no gas
+              required."
+              values={{
+                eip: (
+                  <Link to="https://eips.ethereum.org/EIPS/eip-4895">
+                    <FormattedMessage defaultMessage="EIP-4895" />
+                  </Link>
+                ),
+              }}
+            />
+          </Text>
+        </section>
+
+        <section>
+          <Anchor to="#enabling-withdrawals" id="enabling-withdrawals">
+            <SectionTitle level={2} className="mb10">
+              <FormattedMessage defaultMessage="Enabling withdrawals" />
+            </SectionTitle>
+          </Anchor>
+          <Text className="mb10">
+            <FormattedMessage
+              defaultMessage="For your validator account to be capable of withdrawals of any
+              kind, a withdrawal address must be provided and registered to your account on the
+              Beacon Chain. This should be an address you control, and cannot be changed once set."
+            />
+          </Text>
+
+          <WithdrawalsTabComparison />
+        </section>
+
+        <section>
+          <Anchor to="#upgrade-your-keys" id="upgrade-your-keys">
+            <SectionTitle level={2} className="mb10">
+              <FormattedMessage defaultMessage="Upgrade your keys: BLS to execution change" />
+            </SectionTitle>
+          </Anchor>
+          <Text className="mb10">
+            <FormattedMessage
+              defaultMessage="If your validator account still has BLS withdrawal credentials ({type0}),
+              a one-time message must be broadcast signaling which execution layer account should be
+              used for all withdrawals."
+              values={{ type0: <Code>0x00</Code> }}
+            />
+          </Text>
+
+          <section>
+            <Heading level={3} className="mb10">
+              <FormattedMessage defaultMessage="Available tools" />
+            </Heading>
+            <ul>
+              <li>
+                <strong>ethdo</strong> -{' '}
+                <Link
+                  inline
+                  primary
+                  to="https://github.com/wealdtech/ethdo/blob/master/docs/changingwithdrawalcredentials.md"
+                >
+                  Changing withdrawal credentials
+                </Link>
+              </li>
+              <li>
+                <strong>Staking Deposit CLI</strong> -{' '}
+                <Link inline primary to="https://notes.ethereum.org/@hww/btec">
```

Comment:
Moved to https://notes.ethereum.org/@launchpad/btec

```suggestion
                <Link inline primary to="https://notes.ethereum.org/@launchpad/btec">
```